### PR TITLE
update API docs [Fixes #116365903]

### PR DESCRIPTION
**Pull request by Maroo-b**
In progress: check and update API docs
- fix user show API bug when user have pending friendship requests or friends list

### CODE REVIEW

File: app/api-doc/api/v1/languages_doc.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
 module Api::V1::LanguagesDoc
   extend Apipie::DSL::Concern
 
+
+  def_param_group :language do
+    param :language, Hash, :action_aware => true, :required => true do
```
Comment (tochman) [inline app/api-doc/api/v1/languages_doc.rb]: Try to use the new hash format in the code:

```
param :language, Hash, action_aware: true, required: true do
```

Comment (Maroo-b) [inline app/api-doc/api/v1/languages_doc.rb]: Sure I'll make the changes.

**Comment by Maroo-b** [comment]
@tochman I updated the PR but it seems the build is always failing

**Comment by tochman** [comment]
It fails on phantomjs install randomly. Passed after rebuild.

**Pull request merged**